github-action/src/main.ts

```diff
@@ -138,6 +138,54 @@ export async function runMain(): Promise<void> {
 			return;
 		}
 
+		// If we have a platform specified and the image was built, get the image digest
+		if (buildResult.outcome === 'success') {
+			// Create a digests object to track digests for each platform
+			const digestsObj: Record<string, string> = {};
+			
+			if (platform) {
+				// Extract the image digest from the build output
+				if (buildResult.imageDigest) {
+					core.info(`Image digest for ${platform}: ${buildResult.imageDigest}`);
+					digestsObj[platform] = buildResult.imageDigest;
```

Comment:
Getting an error here that `buildResult.imageDigest` cannot be assigned to `digestsObj[platform]` because `buildResult.imageDigest` is a `Record<string, string>` and you're trying to assign it to a `string` value.